wixcode-users/wix-users.service.json

```diff
@@ -543,6 +604,52 @@
               {  } },
         "extra":
           {  } },
+      { "name": "onConsentPolicyChanged",
+        "labels": [],
+        "nameParams": [],
+        "params":
+          [ { "name": "handler",
+              "type": "wix-users.ConsentPolicyChangedHandler",
+              "doc": "The name of the function or\n the function expression to run when the consent policy is changed." } ],
+        "ret":
+          { "type": "void" },
+        "locations":
+          [ { "lineno": 621,
+              "filename": "users.js" } ],
+        "docs":
+          { "summary": "Sets the function that runs when a user's consent policy was changed using [`setConsentPolicy()`](#setConsentPolicy) or if it was reset.",
```

Comment:
Set shows the function and reset doesn't
Maybe use "if set or reset"
or
 "was changed using `setConsentPolicy` or `resetConsentPolicy`
Also below